cmd/serverless-init/cloudservice/cloudrun.go

```diff
@@ -14,60 +14,68 @@ import (
 )
 
 const (
-	//nolint:revive // TODO(SERV) Fix revive linter
 	revisionNameEnvVar      = "K_REVISION"
 	ServiceNameEnvVar       = "K_SERVICE" // ServiceNameEnvVar is also used in the trace package
 	configurationNameEnvVar = "K_CONFIGURATION"
 	functionTypeEnvVar      = "FUNCTION_SIGNATURE_TYPE"
 	functionTargetEnvVar    = "FUNCTION_TARGET" // exists as a cloudrunfunction env var for all runtimes except Go
 )
 
+const (
```

Comment:
nit: this can probably just move to the other constants. Differentiating them with a comment is sufficient.